dotCMS/src/integration-test/java/com/dotcms/MainSuite.java

```diff
@@ -532,6 +535,10 @@
         Task220203RemoveFolderInodeConstraintTest.class,
         Task220214AddOwnerAndIDateToFolderTableTest.class,
         Task220215MigrateDataFromInodeToFolderTest.class,
+        ContentDispositionFileNameParserTest.class,
+        SecureFileValidatorTest.class,
+        BoundedBufferedReaderTest.class,
+        Task220404RemoveCalendarReminderTest.class,
```

Comment:
This merge is not correct. You're including tests that already existed in MainSuite. Beware, the order is important.